examples/suite/basic_performance_long_with_workers.rb

```diff
@@ -0,0 +1,21 @@
+# format description:
+# each test can be executed by either target duration using :time => N secs
+# or by number of events with :events => N
+#
+# you can specify the number of filter worker threads for each test with :workers => N
+# if you don't specify workers, it defaults to 1
+#
+#[
+#  {:name => "simple json out", :config => "config/simple_json_out.conf", :input => "input/simple_10.txt", :time => 30},
+#  {:workers => 2, :name => "simple json out", :config => "config/simple_json_out.conf", :input => "input/simple_10.txt", :events => 50000},
+#]
+#
+[
+  {:workers => 2, :name => "simple line in/out", :config => "config/simple.conf", :input => "input/simple_10.txt", :time => 120},
```

Comment:
There's no real reason why the examples all use 2 workers. It's just more than one and most machines can easily handle 2 threads so I thought it was a decent example. I'm not opposed to changing it up though.

Yeah, you're right, we probably don't need special examples. Maybe just removing these with_workers examples all together and just changing a couple of the tests in the original example files to use workers.